internal/server/models/organization.go

```diff
@@ -28,6 +28,7 @@ func (o *Organization) ToAPI() *api.Organization {
 }
 
 var domainNameReplacer = regexp.MustCompile(`[^\da-zA-Z-]`)
+var HostDomain = "infrahq.com" // TODO: configure from settings
```

Comment:
need to figure out what to do here before merge

Reply:
Any ideas other than a server setting? Feels like it should be required.

Reply:
I think we should get it into the config, ideally required